src/main/java/de/tum/in/www1/artemis/web/rest/errors/EntityNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package de.tum.in.www1.artemis.web.rest.errors;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
```

Comment:
Is this not available somewhere in the Spring framework? Do we really need to implement this?
(I thought we want to discuss this first so that all REST calls use a consistent mechanism instead of handling it differently)

Reply:
To my knowledge spring does not provide exceptions for this with included error code mapping. I found multiple occasions in Artemis where custom Exceptions are beeing thrown instead of creating own ResponseEntities. Since it seams to be a standard way of handeling errors and provides a few benefits i implemented it for the modeling assessment controller but forgot to mention it in our meeting.